src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -80,38 +93,246 @@ export default function FacilityOrganizationIndex({
+            </TableCell>
+          )}
+        </TableRow>
+        {expandedRows[org.id] &&
+          children.map((child) => (
+            <OrganizationRow
+              key={child.id}
+              org={child}
+              expandedRows={expandedRows}
+              toggleRow={toggleRow}
+              getChildren={getChildren}
+              indent={indent + 1}
+            />
+          ))}
+      </>
+    );
+  };
   return (
     <Page title={t("departments")} hideTitleOnPage={true}>
-      <div className="flex justify-center md:justify-end mt-2 mb-4">
-        <CreateFacilityOrganizationSheet facilityId={facilityId} />
+      <h3 className="mb-4 text-black">{t("departments")}</h3>
+      <div className="flex justify-between items-center mb-4 gap-5 lg:flex-row flex-col-reverse ">
+        <div className="lg:w-3/4 w-full">
+          <Input
+            className="px-2 placeholder:text-xs placeholder:text-gray-500"
+            placeholder={t("filter_by_department_or_team_name")}
+          ></Input>
+        </div>
+        <div className="flex lg:justify-end w-full">
+          <CreateFacilityOrganizationSheet facilityId={facilityId} />
+        </div>
       </div>
-      <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 pb-4">
-        {data.results.map((org: FacilityOrganization) => (
-          <Card key={org.id} className="relative group">
-            <CardHeader>
-              <div className="flex items-start justify-between">
-                <CardTitle className="flex items-center gap-2">
-                  {org.name}
-                </CardTitle>
-                <CardDescription>{org.org_type}</CardDescription>
-              </div>
-            </CardHeader>
-
-            <CardFooter>
-              <Button variant="outline" asChild className="w-full">
-                <Link
-                  href={`/departments/${org.id}`}
-                  className="flex items-center justify-center gap-2"
-                >
-                  {t("view_details")}
-                  <CareIcon icon="l-arrow-right" className="h-4 w-4" />
-                </Link>
-              </Button>
-            </CardFooter>
-          </Card>
-        ))}
+      <div className="flex-row items-center flex gap-3 text-blue-800 text-xs sm:text-sm border-2 rounded-lg border-blue-200 bg-blue-50 p-4 mb-4">
+        <div className="sm:p-2 p-1 bg-blue-100 rounded-sm">
+          <CareIcon icon="l-info-circle" className="h-6 w-6 text-blue-900" />
+        </div>
+        <div className="">
+          <p className="">
+            {t("click")}{" "}
+            <span className="font-semibold">{t("add_department_team")}</span>{" "}
+            {t("to_create_new_one")}
+          </p>
+          <p className="">
+            {t("click")}{" "}
+            <span className="font-semibold">{t("see_details")}</span>{" "}
+            <CareIcon icon="l-arrow-up-right"></CareIcon>{" "}
+            {t("to_manage_create_more")}
+          </p>
```

Comment:
let's not do word by word translations; this will be incorrect in other languages as placement of words is not exactly as same as how it's placed in english; use `<Trans />` block to achieve translations with stylings. Checkout similar usages in the codebase.